### DISCUSSION TITLE
2024 Q1 Community Manager Proposal

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#59] [Show and tell]
# **1. Objectives & Actions**

### 1.1 Marketing

- DCDAO Twitter Followers reach 1500
- DCDAO’s Media Partners reach 10

### 1.2 DAO Development

- Create a proposal about the DCDAO Constitution
- Improve DCDAO awareness
- Grow DCDAO Membership into 14

### 1.3 Community

- Sort the community member roles and maintain public order in the community
- Interact with community members through regular events(quiz games, voice calls)
- Grow the Discord member to 2900, and telegram group users into 9500

# 2. Funding Support

The funding support is approximately 5000 USD/month. Here are the details.

## 2.1 Ambassador Program Monthly Expense

Within 2000USDT-3000USDT/month(the rewards will be paid by Ring)

## 2.2 Community Manager Monthly Salary

1700 USD

My wallet address： 0x7C548fe9aEd9F6C50cD372196eA5510ac56dBa8F（eth)

## 2.3 Other Expenses

To activate our community and promote DCDAO, I plan to do some events like giveaways or other types of interaction with members in Q1.

Budget: 180,000 Ring for 2024 Q1

DCDAO Marketing Multi-sig Address:

ETH:0x7AC72B45E72dAa1CA2a26FB543E7B096570d1BCd

Polygon:0xbaCd8aBCf84a1ED6B4daB676489f420163e3625b

**Comment by 2xJDubs** [comment]
Due to Cindys departure the Ambassadors of DCDAO will be taking over management of the Marketing Multi-sig.  The Polygon address is staying the same but we will be adding a an Arbitrum Multi-sig as well.  I will create a vote on Snapshot to make this change.
